docs/diagrams/orchestration/auth-only/auth-only.mmd

```diff
@@ -12,11 +13,12 @@ sequenceDiagram
     auth ->> orch_api : Token response containing access token
     orch_api ->> auth : Userinfo request containing access token to /userinfo (UserInfoHandler.java)
     auth ->> orch_api : Userinfo response
+    orch_api <<->> ais : Check for interventions
```

Comment:
This symbol `<<->>` doesn't render properly for me locally

Reply:
I made them all a dotted single ended line, the double-ender doesn't render in markdown <11.